### PR TITLE
Con 2610 epao assessor confirmation

### PR DESCRIPTION
**Pull request by kola-samuel**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Application/Handlers/Apply/CreateApplicationHandler.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ public async Task<Guid> Handle(CreateApplicationRequest request, CancellationTok
 
             return Guid.Empty;
         }
-
+
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Application/Handlers/Apply/CreateApplicationHandler.cs:71]: Remove this - blank lines are ok but not spaces

File: src/SFA.DAS.AssessorService.ApplyTypes/Application.cs [hunk 2]
```diff
@@ -130,4 +130,5 @@ public class Submission
         public Guid SubmittedBy { get; set; }
         public string SubmittedByEmail { get; set; }
     }
+
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.ApplyTypes/Application.cs:133]: Unnecessary change

File: src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs [hunk 3]
```diff
@@ -734,6 +739,21 @@ public async Task<IActionResult> Feedback(Guid id)
             return View("~/Views/Application/Feedback.cshtml", sequenceVm);
         }
 
+        private ApplicationSubmissionType GetApplicationSubmissionTypeAsync(ApplicationResponse application, Sequence sequence)
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs:742]: There is already an ApplicationTypes class which has definitions for types of application StandardWithdrawal etc - so we should not have something new for that.

Other views e.g. 'Feedback.cshtml' are just using the ApplyConst directly - so that would be more consistent if you put the SequenceNo in the view model and the test against the ApplyConst which is also used in dozens of places, so it needs to kept consistent

File: src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml [hunk 4]
```diff
@@ -1,56 +1,139 @@
...
+
+
+                    <h2 class="govuk-heading-m govuk-!-margin-top-6">
+                        What happens next
+                    </h2>
+
+
+                    @if (Model.ApplicationSubmissionType == ApplicationSubmissionType.WithdrawalFromRegister)
+                    {
+                        <p class="govuk-body">
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml:41]: This part looks the same below so should not be duplicated

File: src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs [hunk 5]
```diff
@@ -25,6 +25,7 @@
 using SFA.DAS.AssessorService.Web.ViewModels.Apply;
 using SFA.DAS.QnA.Api.Types;
 using SFA.DAS.QnA.Api.Types.Page;
+using AssessorServiceApplyType = SFA.DAS.AssessorService.ApplyTypes;
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs:28]: Is this used?

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AssessorService.Application/Handlers/Apply/CreateApplicationHandler.cs (hunk 1 above)
Comment (kola-samuel) [reply src/SFA.DAS.AssessorService.Application/Handlers/Apply/CreateApplicationHandler.cs]: removed

File: src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs (hunk 3 above)
Comment (kola-samuel) [reply src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs]: updated

File: src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml (hunk 4 above)
Comment (kola-samuel) [reply src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml:37]: updated

File: src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs (hunk 5 above)
Comment (kola-samuel) [reply src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs]: removed

**Review by ChrisJWoodcock — APPROVED** [review]
